components/abstractions/src/main/java/com/microsoft/kiota/RequestInformation.java

```diff
@@ -473,6 +474,10 @@ private static Object getSanitizedValues(Object value) {
             return ((ValuedEnum) value).getValue();
         } else if (value instanceof UUID) {
             return value.toString();
+        } else if (value instanceof OffsetDateTime) {
+            return ((OffsetDateTime) value).format(DateTimeFormatter.ISO_OFFSET_DATE_TIME);
+        } else if (value instanceof LocalDate) {
```

Comment:
here we'd be missing LocalTime and PeriodAndDuration

Reply:
Yes, I will implement this now that it's been confirmed we want to support these.

Reply:
These are now implemented.